content/en/docs/concepts/secure-plugin-development.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Securely develop plugins for Notation"
+description: "Best practices for securely developing and distributing plugins for Notation"
+type: docs
+weight: 2
+---
+
+## General security guidance
+
+plugins run as a binary on the system, invoked by notation. This means that the plugin has access to the same resources as the user running notation. This includes access to the filesystem, network, and other resources. Ensure you are only accessing resources that are required for your plugin to function.
+
+In some cases, your plugin may have access to sensitive data such as private keys. Ensure that you are not leaking this data to the filesystem, network, or other resources, including `stdout` and `stderr`.
+
+If using the `pluginConfig` key/value store, do not store sensitive data in as the data is not encrypted and is stored as plaintext.
+
```

Comment:
Suggest reminding developers to follow [plugin development Spec](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) strictly when developing a plugin.

Reply:
fixed